client/components/App.jsx

```diff
@@ -3,26 +3,18 @@ import React from 'react';
 import ListingsView from './listingsView.jsx';
 import ReactDOM from 'react-dom';
```

Comment:
This can be removed since you are doing your top level render in the index.jsx file. You have the import statement as needed in that file, but since you aren't doing the final render to the index.html in this file, this import statement can safely be removed.